client/src/elements/schema-editor/schema-editor-wrapper.css

```diff
@@ -1,11 +1,18 @@
 schema-editor-wrapper {
   display: flex;
+  position: relative;
 }
 
 schema-editor-wrapper > :first-child {
   width: 100%;
 }
 
 schema-editor-wrapper > notification {
-  margin-top: 22px; /* this is hacky but is the margin required to align the notification with the input element
+  margin-top: 22px; /* this is hacky but is the margin required to align the notification with the input element */
+  position: absolute;
+  right: 0;
+}
+
+.schema-editor-notification-column {
```

Comment:
could you name this class `schema-editor-input--with-notifications`? Or something even better. The current name suggests that this is the notification-column when it's in fact about making space for the notification column.